app/scripts/views/settings/avatar_camera.js

```diff
@@ -34,6 +34,7 @@ function (_, Cocktail, canvasToBlob, FormView, ProgressIndicator,
   var View = FormView.extend({
     template: Template,
     className: 'avatar-camera',
+    screenName: 'settings.avatar.camera',
```

Comment:
I used the same screen name that *should* be passed in by the router so that we are able to remove these hard coded names in the future.